static/js/views/discovered-addon.js

```diff
@@ -123,6 +129,55 @@ class DiscoveredAddon {
         controlDiv.innerHTML = el;
       });
   }
+  
+  /**
+   * Handle a click on the license button.
+   */
+  handleLicense(e) {
+		if(e.target.getAttribute('data-id')) {
+			const license_url = "https://api.mozilla-iot.org:8443/addons/license/" + e.target.getAttribute('data-id');
```

Comment:
The base URL here needs to be retrieved from the gateway via `API.getAddonsInfo()`. Don't hard code it. Also, you can use a [template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals).